README.md

```diff
@@ -13,11 +13,13 @@ This script depends on having firefox downloaded
 1. Uncompress it `tar -zxvf geckodriver-v0.26.0-macos.tar.gz`
 1. Put geckodriver on the path `export PATH=$PATH:$(pwd)`
 
+## Get credentials in SDR-API
```

Comment:
Can you describe where to do this? Like "on sdr-api stage server"?

Reply:
Doesn't the link below have sufficient information?  Do we need to improve it here or on that other page?

Reply:
Here.